CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
 #### Features
+- Improved Route refresh feature: now it refreshes _closures_ (`List<Closure>`) along with _annotations_ (`LegAnnotation`) and _incidents_ (`List<Incident>`). [#6274](https://github.com/mapbox/mapbox-navigation-android/pull/6274)
```

Comment:
```suggestion
- Improved the route refresh feature to also refresh _closures_ (`RouteLeg#closures`). [#6274](https://github.com/mapbox/mapbox-navigation-android/pull/6274)
```

I'd like to avoid confusing developers that _annotations_ and _incidents_ are also new.